{% extends "single_page_base.html" %}
{% block title %}{{title}}{% endblock %}
{% block nav %}
{% if fastqc %}<li role="presentation"><a href="rendered_data_stat.html">数据统计</a></li>{% endif %}
{% if mapping %}<li role="presentation"><a href="rendered_mapping.html">序列比对</a></li>{% endif %}
<li role="presentation" class="active"><a href="rendered_rseqc.html">数据质控</a></li>
{% if quantification %}<li role="presentation"><a href="rendered_quantitative_analysis.html">定量分析</a></li>{% endif %}
{% if quantification %}<li role="presentation"><a href="rendered_diff_analysis.html">差异分析</a></li>{% endif %}
{% if enrichment %}<li role="presentation"><a href="rendered_enrichment_analysis.html">功能富集</a></li>{% endif %}
{% endblock %}

{% block intro %}
<div class="intro">
  <p>
    <I>我们使用<a href="http://doi.org/10.1093/bioinformatics/bts356" target="_blank">RSeQC</a><a href="#wang">(Wang, L., Wang, S., & Li, W. 2012)</a>，
    对本次测序的数据的文库插入片段，基因区 reads 覆盖度和基因区域 reads 分布3个方面进行评估。
    下面展示了这3项分析的结果。</I>
  </p>
</div>
{% endblock %}

{% block rseqc_inner_distance %}
<div class="part row group_plot">
  <div class="col-md-4">
      <h3>文库插入片段长度分布图</h3>
      <p style="text-indent:32px;font-size:16px" class="haveBg">
        右图展示了样品的插入片段长度的分布情况。其中 x 轴代表文库插入片段长度，
        可以理解为 read1 和 read2在基因组中的距离，
        该值为负数代表 read1 和 read2 有部分重叠 (overlap)。
        (文库片段长度 = 插入片段长度 + read1长度 + read2长度)
      </p>
      <p><a class="btn btn-primary" href="{{inner_distance_plot_dir}}" target="_blank" role="button">更多详情 &raquo;</a></p>
  </div>
  <div class="albumSlider col-md-8" style="margin-left:100px">
    <div class="fullview"><img src="{{inner_distance_plot_path[0]}}" alt="Go barplot"></div>
    <div class="slider">
      <div class="button movebackward" title="向上滚动"></div>
        <div class="imglistwrap">
          <ul class="imglist">
            {% for each_inner_distance_plot_path in inner_distance_plot_path %}
              <li><a href="{{each_inner_distance_plot_path}}" class="example2" title="inner distance plot." rel="example_group3"><img src="{{each_inner_distance_plot_path}}" alt="example2"></a></li>
            {% endfor %}
          </ul>
        </div>
    </div>
  </div>
</div>
{% endblock %}

{% block rseqc_read_duplication %}
<div class="part row group_plot">
  <div class="col-md-4">
      <h3>read duplication plot</h3>
      <p style="text-indent:32px;font-size:16px" class="haveBg">
        Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo,
        tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.
        Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui.
      </p>
      <p><a class="btn btn-primary" href="{{read_duplication_plot_dir}}" target="_blank" role="button">更多详情 &raquo;</a></p>
  </div>
  <div class="albumSlider col-md-8" style="margin-left:100px">
    <div class="fullview"><img src="{{read_duplication_plot_path[0]}}" alt="Go barplot"></div>
    <div class="slider">
      <div class="button movebackward" title="向上滚动"></div>
        <div class="imglistwrap">
          <ul class="imglist">
            {% for each_read_duplication_plot_path in read_duplication_plot_path %}
              <li><a href="{{each_read_duplication_plot_path}}" class="example2" title="read duplication plot." rel="example_group3"><img src="{{each_read_duplication_plot_path}}" alt="example2"></a></li>
            {% endfor %}
          </ul>
        </div>
    </div>
  </div>
</div>
{% endblock %}

{% block rseqc_genebody_coverage %}
<div class="part row group_plot">
  <div class="col-md-4">
      <h3>基因区域 reads 覆盖情况</h3>
      <p style="text-indent:32px;font-size:16px" class="haveBg">
      我们将所有基因的转录本按照比例100等分，分别计算每个区域上的 reads 覆盖情况，
      进行标准化之后得到基因5'端至3'端的整体表达情况。右图展示了基因区域 reads 覆盖的情况。
      通常情况下 reads 在5'端和3'端的覆盖度是一致的，若 reads 覆盖在5'或3'端存在明显的偏好，
      说明样品 RNA可能存在不完整的情况。
      </p>
      <p><a class="btn btn-primary" href="{{genebody_coverage_plot_dir}}" target="_blank" role="button">更多详情 &raquo;</a></p>
  </div>
  <div class="albumSlider col-md-8" style="margin-left:100px">
    <div class="fullview"><img src="{{genebody_coverage_plot_path[0]}}" alt="Go barplot"></div>
    <div class="slider">
      <div class="button movebackward" title="向上滚动"></div>
        <div class="imglistwrap">
          <ul class="imglist">
            {% for each_genebody_coverage_plot_path in genebody_coverage_plot_path %}
              <li><a href="{{each_genebody_coverage_plot_path}}" class="example2" title="genebody coverage plot." rel="example_group3"><img src="{{each_genebody_coverage_plot_path}}" alt="example2"></a></li>
            {% endfor %}
          </ul>
        </div>
    </div>
  </div>
</div>
{% endblock %}

{% block rseqc_read_distrbution %}
<div class="part row group_plot">
  <div class="col-md-4">
      <h3>reads 在基因区域分布</h3>
      <p style="text-indent:32px;font-size:16px" class="haveBg">
        我们统计落在不同基因区域 (包括上下游10kb) 的 reads 的比例。
        结果如右图所示。对于 RNAseq 数据，
        大部分的 reads 应该落在基因的 exon 和 cds 区域。
      </p>
      <p><a class="btn btn-primary" href="{{read_distrbution_plot_dir}}" target="_blank" role="button">更多详情 &raquo;</a></p>
  </div>
  <div class="albumSlider col-md-8" style="margin-left:100px">
    <div class="fullview"><img src="{{read_distrbution_plot_path[0]}}" alt="Go barplot"></div>
    <div class="slider">
      <div class="button movebackward" title="向上滚动"></div>
        <div class="imglistwrap">
          <ul class="imglist">
            {% for each_read_distrbution_plot_path in read_distrbution_plot_path %}
              <li><a href="{{each_read_distrbution_plot_path}}" class="example2" title="read distrbution plot" rel="example_group3"><img src="{{each_read_distrbution_plot_path}}" alt="example2"></a></li>
            {% endfor %}
          </ul>
        </div>
    </div>
  </div>
</div>
{% endblock %}

{% block document_rseqc %}
<h3>参考文献</h3>
<br>
<b id="wang">[1]</b>quality control of RNA-seq experiments. Bioinformatics (Oxford, England), 28(16), 2184–2185
{% endblock %}
